public/assets/js/main.js

```diff
@@ -0,0 +1,9 @@
+var scrollDirection = 1, pageScroll;
+  pageScroll = function() {
+    window.scrollBy(0,scrollDirection);
+    scrolldelay = setTimeout(pageScroll,1);
```

Comment:
I feel it is quite fast so slowing it down might be a good idea.

```suggestion
    scrolldelay = setTimeout(pageScroll,25);
```